packages/rs-dapi-client/src/request_settings.rs

```diff
@@ -77,3 +79,25 @@ pub struct AppliedRequestSettings {
     /// Ban DAPI address if node not responded or responded with error.
     pub ban_failed_address: bool,
 }
+
+impl AppliedRequestSettings {
+    /// Create [AppliedRequestSettings] from [RequestSettings] with default values.
+    ///
+    /// Combine provided [RequestSettings] together with [request-level overrides](TransportRequest::SETTINGS_OVERRIDES).
+    ///
+    ///
+    /// # Arguments
+    ///
+    /// * `global_settings` - global settings for all requests.
+    /// * `request_settings` - settings for a specific request.
+    pub fn from_settings<R: TransportRequest>(
```

Comment:
`From<(&RequestSettings, &RequestSettings)>` ? ))

Reply:
not used, removed.